src/migrations/20181907164000-endpoint-specific-logschannel.js

```diff
@@ -1,8 +1,9 @@
+const Bluebird = require('bluebird');
 const fs = require('fs');
 const configJsonPath = process.env.CONFIG_MOUNT_POINT;
 
-exports.up = function(knex, Promise) {
-	return new Promise(resolve => {
+exports.up = function(knex) {
+	return new Bluebird(resolve => {
```

Comment:
You could keep this as 
```suggestion
	return new Promise(resolve => {
```
and just return a native promise - they're definitely supported in node 12

Reply:
This is for the `.tap` lower down. I could convert these to async/await, but I'd rather leave the existing migrations alone tbh.

Reply:
Ah I couldn't see it, fair enough